### PR TITLE
Moved the Reviewd button into the regular project view and styled it …

### PR DESCRIPTION
**Pull request by wolfravenous**
…to allign well.  This will eliminate the extra click required to access the Reviewed button under edit project settings when doing a weekly review.

### CODE REVIEW

**Comment by C-Otto** [comment]
Could you add a screenshot? I'd love to see the change, but I'm on mobile only for a while. Thank you

**Comment by wolfravenous** [comment]
C-Otto, here are some pngs that show the new placement of the Reviewed button in the regular project view and how it is now stylized to look more like the Update and Cancel buttons that are found under the edit settings display, in that it now has a light blue border upon rollover as they have light green and light red(pink) borders respectively upon rollover.

reviewbuttonplacement
reviewedbuttonrollover
updatebuttonrollover
cancelbuttonrollover

**Comment by C-Otto** [comment]
Great! Thank you

**Comment by C-Otto** [comment]
- The button is too tall to fit into the light gray bar on my screen
- When you click "Edit Project Settings", the button is partially hidden

**Comment by wolfravenous** [comment]
Then you would want to tweak the CSS settings so the button is shorter for
your screen.. I only tested with ff browser.  Also I don't understand your
comment about the button being  partial hidden when you click edit project
settings.  On my system and configuration when I click the edit project
settings option the reviewed button is not visible at all.  That was my
intent to make it visible when viewing a project without having to click
the edit project settings.  Seems like any issues you have can be solved
with CSS tweaks...  The thing I was really after was getting the
functionality of the reviewed button out from underneath the project
settings and in the regular project view, which I accomplished.

With Best Regards
Steve Hampton

**Comment by C-Otto** [comment]
I am very grateful for your application and I am using it daily on my tracks installation. However, before a merge of this PR I'd like the CSS tweaks you mentioned to be implemented. If you want to do this, I'd be even more grateful - but I'm happy to just complain/mention this until someone feels the burning motivation to help :)

**Comment by wolfravenous** [comment]
I will work on putting together a CSS implementation that appears decent
across multiple browsers, will probably get it done over Christmas break
unless some other CSS wizard wants to do it cause CSS is not my strong
point but I will struggle through it if need be :)

With Best Regards
Steve Hampton

**Comment by wolfravenous** [comment]
C-Otto,
Can you please post a capture of what you are seeing on your end that is a problem in the CSS?  I am posting below Grabs from Safari, Firefox and Chrome, and while they are not exactly identical I don't see in any of those browsers the problem you are referring to with "The button is too tall to fit into the light gray bar on my screen"  Also, what browser and platform configuration are you using? Tablet, Desktop, Mobile?  I wanted to try to eliminate the CSS issues this week but first I have to replicate them :)

chromegrab
firefoxgrab
safarigrab

**Comment by C-Otto** [comment]
screenshot

Desktop

Google Chrome	55.0.2883.87 (Official Build) (64-bit)
Revision	ad0be09aa3ca814168d079b52825f6f80e22f0e8-refs/branch-heads/2883@{#723}
OS	Linux

File: app/assets/stylesheets/legacy.scss [hunk 1]
```diff
@@ -1157,11 +1157,11 @@ http://particletree.com/features/rediscovering-the-button-element/ */
...
     float: left;
-    margin:0px 7px 0 0;
+    margin:5px 10px 0 0;
     background-color:#f5f5f5;
-    border:1px solid #dedede;
-    border-top:1px solid #eee;
-    border-left:1px solid #eee;
+    border:1px solid #999;
+    border-top:1px solid #ccc;
+    border-left:1px solid #ccc;
```
Comment (mattr-) [inline app/assets/stylesheets/legacy.scss]: These changes to the margin and the border will have a global effect. If these are only for the reviewed button, could you please create a separate class for that and use these colors there?

File: app/assets/stylesheets/legacy.scss [hunk 2]
```diff
@@ -1170,17 +1170,17 @@ http://particletree.com/features/rediscovering-the-button-element/ */
     font-weight:bold;
     color:#565656;
     cursor:pointer;
-    padding:3px 5px 7px 5px; /* Links */
+    padding:2px 5px 3px 5px; /* Links */
```
Comment (mattr-) [inline app/assets/stylesheets/legacy.scss]: Same theme as above.

**Comment by C-Otto** [comment]
@wolfravenous ping? I'd love to see this merged :)

**Pull request closed without merging**